<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZigBee DEV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Devices informations</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>NONE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Rafraichi toutes les information du module connecté par RS232</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>491</width>
      <height>421</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>1</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="toolTip">
        <string>Adresse MAC du module</string>
       </property>
       <property name="text">
        <string>@MAC :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="toolTip">
        <string>Numero du reseaux ZigBee</string>
       </property>
       <property name="text">
        <string>Pane ID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_5"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="toolTip">
        <string>Masque des canneaux de transmission
00000800 a 07FFF800 pour 2,4GHz</string>
       </property>
       <property name="text">
        <string>Channel mask :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_6">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>00100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>00040000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>00000800</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="toolTip">
        <string>Le role de module dans l'architecture reseaux.
Coordinator
Router
End Device</string>
       </property>
       <property name="text">
        <string>Role :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coordinator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Router</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>End device</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Security</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_17">
       <property name="toolTip">
        <string>Adresse ip du module</string>
       </property>
       <property name="text">
        <string>@IP:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>194</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="toolTip">
        <string>Routeur et End Device -&gt; Rejoindre le réseau.
Coordinator -&gt; Creer le réseau.</string>
       </property>
       <property name="text">
        <string>Join</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="toolTip">
        <string>Routeur et End Device -&gt; se deconnecter du réseau.
Coordinator -&gt; Supprimer le réseau.</string>
       </property>
       <property name="text">
        <string>Leave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="toolTip">
        <string>Routeur et End Device -&gt; Connection dès que le réseau est disponible.
Coordinator -&gt; Creer le réseau dés sa mise sous tension.</string>
       </property>
       <property name="text">
        <string>Connection automatique</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>581</width>
      <height>137</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <property name="toolTip">
        <string>liste des modules connectés sur le réseau</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="toolTip">
        <string>Data a transmettre.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSlider">
       <property name="toolTip">
        <string>Reglage de la puissance de transmission</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="toolTip">
        <string>Send à la node selectionnée</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_9">
       <property name="toolTip">
        <string>Send à toutes les nodes sur le réseau</string>
       </property>
       <property name="text">
        <string>Send to All</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="toolTip">
        <string>Rafraichir la liste des modules connectés sur le réseau</string>
       </property>
       <property name="text">
        <string>Refresh node</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>Puissance de transmission</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TX power level dBm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="toolTip">
        <string>Ping la node selectionnée</string>
       </property>
       <property name="text">
        <string>Ping</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox_com_port"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="RefreshpushButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="closePort">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="connecterPort">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="SendpushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Remote</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>mot de passe en hexa pour une commande a distance.
Min: 00000000 Max: FFFFFFFF
ex : 12345678</string>
        </property>
        <property name="text">
         <string>Own pass :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="toolTip">
         <string>mot de passe en hexa pour une commande a distance.
Min: 00000000 Max: FFFFFFFF
ex : 12345678</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>mot de passe en hexa de la node selectionnée pour une commande a distance.
Min: 00000000 Max: FFFFFFFF
ex : 12345678</string>
        </property>
        <property name="text">
         <string>Node pass:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="toolTip">
         <string>mot de passe en hexa de la node selectionnée pour une commande a distance.
Min: 00000000 Max: FFFFFFFF
ex : 12345678</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_10">
        <property name="toolTip">
         <string>Valide le nouveau mot de passe</string>
        </property>
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Commande AT à transmettre à la node selectionée</string>
        </property>
        <property name="text">
         <string>AT cmd:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_7">
        <property name="toolTip">
         <string>Commande AT à transmettre à la node selectionée</string>
        </property>
        <item>
         <property name="text">
          <string>IO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+WRSSI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+WLQI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+WSRC?</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="toolTip">
         <string>Transmettre la requête</string>
        </property>
        <property name="text">
         <string>send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Link Quality Indicator</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="toolTip">
         <string>Qualité du reseaux entre le module connecté en RS232 et la Node selectionnée.
0 - Bad
255 - good</string>
        </property>
        <property name="text">
         <string>LQI:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_12">
        <property name="toolTip">
         <string>Qualité du reseaux entre le module connecté en RS232 et la Node selectionnée.
0 - Bad
255 - good</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="toolTip">
         <string>puissance du reseaux entre le module connecté en RS232 et la Node selectionnée.</string>
        </property>
        <property name="text">
         <string>RSSI:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_14">
        <property name="toolTip">
         <string>puissance du reseaux entre le module connecté en RS232 et la Node selectionnée.</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_15">
        <property name="toolTip">
         <string>puissance du reseaux entre le module connecté en RS232 et la Node selectionnée.</string>
        </property>
        <property name="text">
         <string>dBm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_16">
        <property name="toolTip">
         <string>Qualité du reseaux entre le module connecté en RS232 et la Node selectionnée.
0 - Bad
255 - good</string>
        </property>
        <property name="text">
         <string>0-255</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>130</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Sleep(ms)</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>164</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox">
        <property name="toolTip">
         <string>temps en ms ou le module sleep</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_13">
        <property name="toolTip">
         <string>reglage du sleep intervals</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="toolTip">
         <string>temps en ms ou le module est activé</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>sleep time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>active time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_14">
        <property name="toolTip">
         <string>lancement du mode Sleep</string>
        </property>
        <property name="text">
         <string>Sleep</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
